spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe ReviewsController do
+  describe "POST create" do
+    context "with authenticated user" do
+      before do
+        @user = Fabricate(:user)
```

Comment:
You should be using `let` instead of instance variables in your specs. Instance variables carry the changes you make to them throughout your specs. `let` variables will reset before each spec. This will save you from bugs in your specs.